core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -72,8 +72,6 @@ object Runtime {
 sealed trait RuntimeStatus extends EnumEntry
 object RuntimeStatus extends Enum[RuntimeStatus] {
```

Comment:
`RuntimeStatus`es represent statuses internal to Leonardo and `{Dataproc,Instance,GceInstance}Statuses` are determined by Google? If so, can we note that in a comment somewhere?